subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskDefinitionIntegrationSpec.groovy

```diff
@@ -432,45 +303,26 @@ public class CustomPlugin implements Plugin<Project> {
         result.output.contains("got it")
     }
 
-    def "can construct a task with @Inject services and constructor args from a plugin"() {
+    @Requires([KOTLIN_SCRIPT])
+    def "can construct a task with @Inject services and constructor args via Project extension function"() {
```

Comment:
How about `via Kotlin friendly DSL`? Then we won't need to change the test name when the feature hits kotlin DSL?

Reply:
done